packages/ketcher-core/src/application/render/restruct/rebond.ts

```diff
@@ -651,24 +657,35 @@ function getBondPath(
       );
       break;
     case Bond.PATTERN.TYPE.DOUBLE_OR_AROMATIC:
+      newHalfBonds = util.updateHalfBondCoordinates(hb1, hb2, -1);
       path = getBondAromaticPath(
         render,
-        hb1,
-        hb2,
+        newHalfBonds[0],
+        newHalfBonds[1],
         bond,
         shiftA,
         shiftB,
         isSnapping,
       );
       break;
     case Bond.PATTERN.TYPE.ANY:
-      path = draw.bondAny(render.paper, hb1, hb2, render.options, isSnapping);
+      console.log(hb1.p, hb2.p);
+
+      newHalfBonds = util.updateHalfBondCoordinates(hb1, hb2, -1);
+      path = draw.bondAny(
+        render.paper,
+        newHalfBonds[0],
+        newHalfBonds[1],
+        render.options,
+        isSnapping,
+      );
       break;
     case Bond.PATTERN.TYPE.HYDROGEN:
+      newHalfBonds = util.updateHalfBondCoordinates(hb1, hb2, 1);
```

Comment:
maybe we could create meaningful constants for 1, -1 instead of numbers here

Reply:
We have a rule https://eslint.org/docs/latest/rules/no-case-declarations 
that does not allow declare variables inside the case statement

Reply:
yes, but could we declare variables elsewhere

Reply:
done